platform/broadcom/sai.mk

```diff
@@ -1,8 +1,9 @@
-LIBSAIBCM_XGS_VERSION = 7.1.32.4
-LIBSAIBCM_DNX_VERSION = 7.1.36.4-2
-LIBSAIBCM_BRANCH_NAME = REL_7.0_202205
-LIBSAIBCM_XGS_URL_PREFIX = "https://sonicstorage.blob.core.windows.net/public/sai/bcmsai/$(LIBSAIBCM_BRANCH_NAME)/$(LIBSAIBCM_XGS_VERSION)"
-LIBSAIBCM_DNX_URL_PREFIX = "https://sonicstorage.blob.core.windows.net/public/sai/bcmsai/$(LIBSAIBCM_BRANCH_NAME)/$(LIBSAIBCM_DNX_VERSION)"
+LIBSAIBCM_XGS_VERSION = 7.1.36.4
+LIBSAIBCM_DNX_VERSION = 7.1.32.4-1
```

Comment:
7.1.32.4-1?
really need to downgrade the version for DNX chips?

Reply:
fixed DNX version